conda_tutorial.md

```diff
@@ -0,0 +1,184 @@
+
+Inside a new Conda installation, the root environment is activated by default, so you can use it without activation.
+
+In other cases, if you want to use an environment (for instance manage packages, or run Python scripts inside it) you need to first activate it.
+
+```
+source activate mynewenv
+```
+The command prompt changes upon the environment’s activation, it now contains the active environment’s name.
+
+The directories of the active environment’s executable files are added to the system path (this means that you can now access them more easily). You can leave an environment with this command:
+
+```
+source deactivate
+```
+
+It needs to be mentioned that upon deactivating an environment, the root environment becomes active automatically.
+
+
+## What are Conda channels?
+
+Channels are the locations of the repositories where Conda looks for packages. Upon Conda’s installation, Continuum’s (Conda’s developer) channels are set by default, so without any further modification, these are the locations where your Conda will start searching for packages.
+
+Channels exist in a hierarchical order. The channel with the highest priority is the first one that Conda checks, looking for the package you asked for. You can change this order, and also add channels to it (and set their priority as well).
+
+![](_static/conda4.png)
+
+If multiple channels contain a package, and one channel contains a newer version than the other one, the channels’ hierarchical order determines which one of these two versions are going to be installed, even if the higher priority channel contains the older version.
+
+![](_static/conda5.png)
+
+
+### Bioconda Channel
+
+See the bioconda paper and the [bioconda web site](http://bioconda.github.io/)
+
+Bioconda is a community-enabled repository of 3,000+ bioinformatics packages, installable via the conda package manager.
+
+
+Add channels
+
+```
+conda config --add channels defaults
+conda config --add channels conda-forge
+conda config --add channels bioconda
+```
+
+Searching, installing and removing packages
+
+To list out all the installed packages in the currently active environment, run:
+
+```
+conda list
+```
+
+Try running a program pre-installed on this instance:
+
+```
+fastqc
+```
+
+To search for all the available versions of a certain package, you can use the search command. For instance, to list out all the versions of the seaborn package (it is a tool for data visualization), run:
+
+```
+conda search -f checkm-genome
+```
+
+Similarly to the conda listcommand, this one results in a list of the matching package names, versions, and channels:
+
+NEED screenshot
+
+To install a package (for instanceseaborn) that is inside a channel that is on your channel list, run this command (if you don’t specify which version you want, it’ll automatically install the latest available version from the highest priority channel):
+
+```
+conda install checkm-genome
+```
+
+You can also specify the package’s version:
```

Comment:
we should probably add a comment somewhere about why versions are important, and what they are